test/echo_server/core_test.clj

```diff
@@ -1,7 +1,18 @@
 (ns echo_server.core-test
   (:require [clojure.test :refer :all]
-            [echo_server.core :refer :all]))
+            [echo_server.core :refer :all])
+  (:import (java.io ByteArrayOutputStream BufferedWriter OutputStreamWriter)))
 
 (deftest a-test
   (testing "one equals one"
     (is (= 1 1))))
+
+(deftest writer-test
```

Comment:
What happens you write multiple messages sequentially?

Reply:
additional test added to codify expected behavior